.github/workflows/ci-goreleaser.yaml

```
@@ -44,4 +56,6 @@ jobs:
         uses: goreleaser/goreleaser-action@v3
         with:
           version: latest
-          args: --snapshot --rm-dist --timeout 1h
+          args: --snapshot --rm-dist --timeout 120m
```

I feel bad for this suggestion, but isn't "2h" easier to read than "120m"? 

```suggestion
          args: --snapshot --rm-dist --timeout 2h
```

---

done